config/pipeline.yaml

```diff
@@ -773,6 +1039,29 @@ jobs:
     <<: *kbuild-gcc-12-riscv-job
     rules: *build-only-trees-rules
 
+  kbuild-gcc-12-riscv-mainline: &kbuild-gcc-12-riscv-mainline-job
+    <<: *kbuild-gcc-12-riscv-job
+    params: &kbuild-gcc-12-riscv-mainline-params
+      <<: *kbuild-gcc-12-riscv-params
+      fragments:
+        - 'debug'
+        - 'kselftest'
+        - 'tinyconfig'
+    rules:
```

Comment:
From old code;
```
            - blocklist: &riscv_kernel_filter
                kernel: ['v3.', 'v4.4', 'v4.9', 'v4.14']
```
I've enabled riscv builds from v4.15. Does it seem correct?

Reply:
Correct